src/k8s/pkg/k8sd/app/hooks_bootstrap.go

```diff
@@ -319,8 +319,8 @@ func onBootstrapControlPlane(s *state.State, initConfig map[string]string) error
 		return fmt.Errorf("failed to create k8s client: %w", err)
 	}
 
-	if err := client.WaitApiServerReady(s.Context); err != nil {
-		return fmt.Errorf("k8s api server did not become ready in time: %w", err)
+	if client.WaitApiServerReady(s.Context); err != nil {
+		return fmt.Errorf("kube api-server did not become ready in time: %w", err)
```

Comment:
Otherwise you are returning the previous `err`
```suggestion
	if err := client.WaitApiServerReady(s.Context); err != nil {
		return fmt.Errorf("kube api-server did not become ready in time: %w", err)
```